assets/js/aplus.js

```diff
@@ -600,6 +604,47 @@ $(function() {
             if (!csrfSafeMethod(settings.type) && !this.crossDomain) {
                 xhr.setRequestHeader("X-CSRFToken", getCookie('csrftoken'));
             }
+            if (true) {
+                
+                var header = {
+                "alg": "HS256",
+                "typ": "JWT"
+                };
+  
+                var data = {
+                    "course": course_code
+                };
+  
+                var secret = "extremly secret:)";
```

Comment:
If you think about it, this isn't very secret :)

In other words, the JWT token is supposed to be generated where it is secret, i.e., in the server.

Reply:
Yeah thought so, not really sure how/where it should e done